Formula/arangodb.rb

```diff
@@ -93,5 +114,8 @@ def plist; <<~EOS
     testcase = "require('@arangodb').print('it works!')"
     output = shell_output("#{bin}/arangosh --server.password \"\" --javascript.execute-string \"#{testcase}\"")
     assert_equal "it works!", output.chomp
+
+    output = shell_output("#{bin}/arangodb --version")
```

Comment:
Can we make it test anything else than the version that tests what the program can do other than print a number ?

Reply:
I've added a start/stop test